pyqttoolkit/services/module_service.py

```diff
@@ -111,7 +111,9 @@ def _create_root_module(self, id_):
         return window
 
     def _create_sub_module(self, id_):
-        window = ModuleWindow(self._theme_manager, id_)
+        title = self._project_manager.get_module_window_title(id_)\
```

Comment:
Wouldn't this work better on the module itself? So it would be `self._registered_modules[id_].get_module_window_title()`. You could even provide an empty implementation on the base class, returning `None`.

You can then get any other services you require through dependency injection in the module object.

Reply:
The `Module` is not the `ModuleWindow` though. How can I access the `ModuleWindow` object from the `Module`, as the module service encapsulates this link

Reply:
The `ModuleService` would still create the `ModuleWindow`. The `Module` would just provide the `ModuleService` with the title, instead of the `project_manager` giving the `ModuleService` the title